.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
```

Comment:
What's the difference between "name" and "title"?
Suggest to change "name" to "category".

Reply:
That table with keywords is actually autogenerated via github GUI, I wouldn't change that for now.